workflows/temporal/temporal.go

```diff
@@ -133,6 +133,16 @@ func (c *TemporalWF) Close() {
 	c.client.Close()
 }
 
+func (c *TemporalWF) Pause(ctx context.Context, req *workflows.WorkflowReference) error {
+	// Unimplemented
+	return nil
```

Comment:
Should this return an error? Same for Resume below? Otherwise, how would the user know the operation failed?